ThreeDSecure/src/main/java/com/braintreepayments/api/CardinalClient.java

```diff
@@ -48,12 +49,12 @@ public void onValidated(ValidateResponse validateResponse, String serverJWT) {
         }
     }
 
-    void continueLookup(FragmentActivity activity, ThreeDSecureResult threeDSecureResult, CardinalValidateReceiver validateReceiver) throws BraintreeException {
+    void continueLookup(ThreeDSecureResult threeDSecureResult, CardinalChallengeObserver challengeObserver) throws BraintreeException {
         ThreeDSecureLookup lookup = threeDSecureResult.getLookup();
         String transactionId = lookup.getTransactionId();
         String paReq = lookup.getPareq();
         try {
-            Cardinal.getInstance().cca_continue(transactionId, paReq, activity, validateReceiver);
+            Cardinal.getInstance().cca_continue(transactionId, paReq, challengeObserver);
```

Comment:
This does feel like it'll change in the near future after we coordinate with Cardinal.